### ISSUE TITLE
Implement validation for messages with associated schema

### ISSUE THREAD

**Issue by shouples** [confluentinc/vscode#576]

**Comment by shouples** [comment]
On hold until https://github.com/confluentinc/ide-sidecar/issues/159 is done

**Comment by rohitsanj** [comment]
@shouples https://github.com/confluentinc/ide-sidecar/issues/159 is done and this is now unblocked.
